common-model/src/main/java/com/sequenceiq/common/api/type/CdpResourceType.java

```diff
@@ -0,0 +1,17 @@
+package com.sequenceiq.common.api.type;
+
+public enum CdpResourceType {
+    DATAHUB,
+    DATALAKE,
+
+    DEFAULT;
+
+    public static CdpResourceType fromStackType(String type) {
+        if (type.toLowerCase().contains("ditrox") || type.toLowerCase().contains("workload")) {
+            return DATAHUB;
+        } else if (type.toLowerCase().contains("sdx") || type.toLowerCase().contains("datalake")) {
+            return DATALAKE;
+        }
+        return DEFAULT;
```

Comment:
what is a default resource type? just askin'. if there is no such type, maybe we should handle it as an unidentified type with some exception

Reply:
like freeipa etc..

Reply:
Ah I see!